### ISSUE TITLE
Does `handleError` wrap all possible authentication errors in `v0.5`?

### ISSUE THREAD

**Issue by steve-marmalade** [awinogrodzki/next-firebase-auth-edge#30]
Hello there, in `v0.5` I now see some middleware crashes like:

```
Error: There is no user record corresponding to the provided identifier.
    at (node_modules/next-firebase-auth-edge/lib/auth/index.js:69:0)
    at (node_modules/next-firebase-auth-edge/lib/auth/index.js:111:0)
    at (node_modules/next-firebase-auth-edge/lib/next/middleware.js:77:38) {
  errorInfo: {
  code: 'auth/user-not-found',
  message: 'There is no user record corresponding to the provided identifier.'
},
  prefix: 'auth'
}
```

The error itself is reasonable -- and I expected the `handleError` function I provided to be called so that I could handle the error similar to the support you added in #9 . In the logs, it does not look like my `handleError` function is being called. Is it possible that the above error is not being caught by the error handler?

**Comment by awinogrodzki** [comment]
Hey @steve-marmalade! Thanks for reporting. You are right, I was not handling errors that pop up during token refresh.

I fixed that already in `0.5.1`: https://github.com/awinogrodzki/next-firebase-auth-edge/commit/ec8085fb03d6394e7fc8f5a03fe69e2762a9ad78

Let me know how that works!

**Comment by steve-marmalade** [comment]
Thanks, working as expected!
